static/chat_db_migrations/1561059285_add_whisper_keys.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE whisper_keys (
+  chat_id TEXT NOT NULL PRIMARY KEY ON CONFLICT IGNORE,
+  key BLOB NOT NULL
+)
```

Comment:
without rowid can be used

Reply:
@cammellos any reason not to? not trying to enforce just curious why?

Reply:
no reason, didn't think of it, but we don't need `rowid` so I have added it now, thanks!